data/_demodata/applications/datacatalogue/Datasets.csv

```diff
@@ -101,5 +101,5 @@ Sepages,base_respi_enfant_20210326,Respiratory_child_data,person,,Table containi
+"testCohort2,test dataset for testCohort2,label for test dataset for cohort 2,medicine dispensation,""distance (traffic),Traffic,IgE (Food),Food,Allergic sensitization,Allergy,Food_IgE,Inhalant_Skin prick test,Inhalant,Grass,Built environment/neighbourhood characteristics,Physical environment,Any inhalant allergic sensitization (IgE),IgE (Inhalant),facilities_richness,main_land_use_agricultural,Chemical exposure,pm25ni,Air pollution,distance (noise),Noise,OCs & PBDEs,Chemicals,Food allergy,Inhalant allergy,Any allergy,IgE - Any food allergic sensitization,Skin prick test - Any food allergic sensitization,Food_Skin prick test,Chicken egg,Cow milk,Fish,Fruit,Rye,Inhalant_IgE,Nut,Peanut,Sesame,Shellfish,Wheat,IgE - Any inhalant allergic sensitization,IgE - House dust mite,Mould,Skin prick test - Any inhalant allergic sensitization,Skin prick test - Cat,Dog,Skin prick test - House dust mite,IgE - Cat,Tree,Anaphylaxis,Cat (Skin prick test),Skin prick test (Inhalant),Pets,Cats (any),main_land_use_continuous_urban_fabric,Cats (number),Dogs (any),Dogs (number),Furry Pets,Housing characteristics,area_level_ses_tertiles,Social context,area_level_ses_quintiles,population_density,building_density100,building_density300,connectivity_density100,connectivity_density300,access_to_public_transport_lines100,access_to_public_transport_lines300,main_land_use_industrial,main_land_use_forests,access_to_public_transport_lines500,access_to_public_transport_stops100,access_to_public_transport_stops300,access_to_public_transport_stops500,main_land_use_medium_urban_fabric,facilities_density,land_use_mix,walkability,main_land_use_airports,main_land_use_other,main_land_use_port,main_land_use_transports,main_land_use_green_urban,main_land_use_low_density,main_land_use_water,Workplace characteristics,House dust mite (Skin prick test),Passive smoking,pm25,pm25fe,Biological exposure,Other physical environment characteristics,Cat (IgE),nox,Meteorology,pm25k,pm25cu,pm25s,Any inhalant allergic sensitization  (Skin prick test),pm25si,pm25v,pm25zn,pm10,pm10si,pm25abs,pm10v,pm10zn,pm10cu,pm10fe,pm10k,pm10ni,pm10s,pmcoarse,mean_temperature,minimum_temperature,maximum_temperature,mean_relative_humidity,minimum_relative_humidity,maximum_relative_humidity,DNA_damage_UV_dose,erythemal_UV_dose,vitamin_d_UV_dose,land_surface_temperature,meteo_distance,blue_distance,Natural spaces,green_distance,blue_size,green_size,blue_access,green_access,ndvi100,ndvi500,ndvi300,lden,lden_categorical,ln_categorical,distance_traffic,traffic_load_all_roads,traffic_density,Unhealthy food environment,traffic_load_major_roads,unhealthy_facilities_density,unhealthy_facilities_density_osm,Phenols,Phthalates,PFASs,Creatinine,Metals,OPs,Cotinine,PAHs,Glycolethers,Other pesticides,Skin prick test (Food),Any food allergic sensitization (IgE),Urticaria,Any food allergic sensitization (Skin prick test),Radiation exposure,no2,House dust mite (IgE),ln"",test description of dataset for cohort 2,83737,1959,1985"
```

Comment:
What was this change for again? Wonder if the extra "" is causing some of the issues?